internal/bodyprocessors/json.go

```diff
@@ -31,7 +34,7 @@ func (js *jsonBodyProcessor) ProcessRequest(reader io.Reader, v plugintypes.Tran
 
 func (js *jsonBodyProcessor) ProcessResponse(reader io.Reader, v plugintypes.TransactionVariables, _ plugintypes.BodyProcessorOptions) error {
 	col := v.ResponseArgs()
-	data, err := readJSON(reader)
+	data, err := readJSON(reader, ResponseBodyRecursionLimit)
```

Comment:
I think it should never be -1 and always force a limit.

Reply:
So we are assuming we don't trust the downstream service neither?

The directive takes into account that the request might be crafted so it creates a DoS of some kind on the body processor (as it happened in ModSecurity). Will the response value be configurable, and different from the request?